### ISSUE TITLE
Search: Remember checkboxes set

### ISSUE THREAD

**Issue by WhiteChairFromIkea** [doublecmd/doublecmd#175]
![21011](https://user-images.githubusercontent.com/46539653/131460640-e207a108-aae8-47ae-8cbb-242690058fab.png)

1. Use search, use "Find data" -> "Find text in file" checkbox;
2. Close search, open search again;
3. "Find data" -> "Find text in file" checkbox is not checked, forgotten.

It is expected for checkbox configuration to be remembered. This would be consistent with "File mask" field: this field is remembered across the sessions.

```
Double Commander
Version: 1.0.0 alpha
Revision: 9891
Commit: 48451b4aa
Build date: 2021/08/26
Lazarus: 2.0.12.0
FPC: 3.2.0
Platform: x86_64-Win64-qt5
OS version: Windows 10 Home 20H2 x86_64

Widgetset library: Qt 5.13.0, libQt5Pas 5.6.1
```

**Comment by alexx2000** [comment]
It is by design. Use "Last search" button to restore last search options.

**Comment by WhiteChairFromIkea** [comment]
I think this is misleading. I already see half of configuration set from last search session: "files mask", "text file string", "search part of filename". Why this particular checkbox would be treated differently?

Maybe add configuration checkbox under "Last search" button, called "Automatically use last search settings"? This would solve my confusion.

**Comment by alexx2000** [comment]
No, this is not only for this checkbox, all search options resets to its defaults ("text file string" stays but disabled).
